build.gradle

```diff
@@ -298,12 +285,7 @@ project('spring-integration-amqp') {
 	description = 'Spring Integration AMQP Support'
 	dependencies {
 		compile project(":spring-integration-core")
-		compile("org.springframework.amqp:spring-rabbit:$springAmqpVersion") {
-			exclude group: 'org.springframework', module: 'spring-aop'
-			exclude group: 'org.springframework', module: 'spring-context'
-			exclude group: 'org.springframework', module: 'spring-core'
-			exclude group: 'org.springframework', module: 'spring-tx'
```

Comment:
Why are you removing all these exclusions? (I don't remember why we added them).

Reply:
We added them to avoid classpath conflicts, mostly in case of IO Platform compatibility tests.
1. Now we don't have them
2. Now we keep all our Spring dependencies aligned with the latest SF.
3. Pay attention how I didn't remove exclusion from the `spring-retry` - it is still based on the SF `4.3.x`.